frontend/lib/src/components/elements/PlotlyChart/PlotlyChart.tsx

```diff
@@ -58,9 +62,10 @@ export interface Selection extends SelectionRange {
   yref: string
 }
 
+// Default height for Plotly charts
 export const DEFAULT_HEIGHT = 450
-const SELECTIONS = "selections"
-const DATA = "data"
+// Minimum width for Plotly charts
```

Comment:
The minimum width? 400 would be a bit to wide

Reply:
I think normal container width is 700 so 400 doesn't seem too bad? Why do u think it's too wide

Reply:
This minimum is mainly for cases where the users resize the window to a very small width. And if the width gets to smaller, altair will start logging warnings/errors. But I don't think we have to limit this even further since 400 would allow a lot of responsiveness.

Reply:
>But I don't think we have to limit this even further since 400 would allow a lot of responsiveness.

You mean 150, not 400 right?